src/ranges.rs

```diff
@@ -0,0 +1,453 @@
+use endianity::{Endianity, EndianBuf};
```

Comment:
It's a good idea, but I think this should be done as a crate wide setting, and it seems that the `#![deny(missing_docs)]` in `lib.rs` is enough.